lib/src/base-styles/_nx-btn.scss

```diff
@@ -20,6 +20,7 @@
   @include container-horizontal;
   @extend %nx-btn--disabled;
 
+  box-sizing: content-box;
```

Comment:
Why this change?

Reply:
> Why this change?

So for one thing, it turns out the UA stylesheet (in Chrome at least) had this set to `border-box` by default for `<button>`s.  I think I changed it to make the the width calculations here and in the mixin slightly simpler. However it also made the min-width calculation in the pagination button more complicated, and I just realized I have that calculation wrong (forgot borders).  So I've now changed this including removing this `box-sizing`. 3568b477